design.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path')
+
+const isDev = process.env.NODE_ENV !== 'production'
+
+const icons = ['ab-testing']
+
+if (isDev) icons.push('check')
+
+const iconsRoot = path.join(
+  path.dirname(require.resolve('@mdi/svg/package.json')),
```

Comment:
Did this package become a `dependency`?

Reply:
It will at some point in https://github.com/oacore/design/pull/209

Reply:
It's better to list this as s `dependency` here since you use `path.resolve` to it in the config.

Reply:
That's config is meant to be removed before merge

Reply:
Sorry, it was clearly said in the commit message, I forgot about. Sure, let's do it in #209.